File: calc.py
import os
def clrscr():
    os.system('cls' if os.name == 'nt' else 'clear')
while True:
    print("menu")
    print("1. add")
    print("2. subtract")
    print("3. multiply")
    print("4. devide")
    choice = input("enter 1-4: ")
    clrscr()
    if choice == '1':
        num1 = input("first number: ")
        num2 = input("second number: ")
        clrscr()
        print('the sum is', int(num1) + int(num2))
    elif choice == '2':
        num1 = input("first number: ")
        num2 = input("second number: ")
        clrscr()
        print('the differance is', int(num1) - int(num2))
    elif choice == '3':
        num1 = input("first number: ")
        num2 = input("second number: ")
        clrscr()
        print('the product is', int(num1) * int(num2))
    elif choice == '4':
        num1 = input("first number: ")
        num2 = input("second number: ")
        clrscr()
        print('the quotient is', float(int(num1)/int(num2)))
    else:
        print("goodbye")
        exit

File: calc.go
package main

import (
	"fmt"
)

func main() {
	var choice string
	fmt.Println("menu")
	fmt.Println("1. add")
	fmt.Println("2. subtract")
	fmt.Println("3. multiply")
	fmt.Println("4. devide")
	fmt.Println("enter options 1-4")
	_, err := fmt.Scanln(&choice)
	if err != nil {
		fmt.Println("error")
	}
	if choice == "1" {
		var num1, num2 float64
		fmt.Println("enter 1st number")
		_, err = fmt.Scanln(&num1)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("enter 2nd number")
		_, err = fmt.Scanln(&num2)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("sum is:", (num1 + num2))
	} else if choice == "2" {
		var num1, num2 float64
		fmt.Println("enter 1st number")
		_, err = fmt.Scanln(&num1)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("enter 2nd number")
		_, err = fmt.Scanln(&num2)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("differance is", (num1 - num2))
	} else if choice == "3" {
		var num1, num2 float64
		fmt.Println("enter 1st number")
		_, err = fmt.Scanln(&num1)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("enter 2nd number")
		_, err = fmt.Scanln(&num2)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("product is:", (num1 * num2))
	} else if choice == "4" {
		var num1, num2 float64
		fmt.Println("enter 1st number")
		_, err = fmt.Scanln(&num1)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("enter 2nd number")
		_, err = fmt.Scanln(&num2)
		if err != nil {
			fmt.Println("error")
		}
		fmt.Println("quotient is:", (num1 / num2))
	}
}
